src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -88,7 +88,8 @@ case class SgeBackend(actorSystem: ActorSystem) extends Backend with SharedFileS
       logger.error(s"Failed to update database status: $status, rc: $rc because $e")
   }
 
-  private def updateSgeJobTable(call: BackendCall, status: String, rc: Option[Int], sgeJobId: Option[Int]): Future[Unit] = {
+  private def updateSgeJobTable(call: BackendCall, status: String, rc: Option[Int], sgeJobId: Option[Int])
+                               (implicit ec: ExecutionContext): Future[Unit] = {
```

Comment:
Hrm I've ripped out these `updateWhatevsBackend` methods with my #415 work, I'll have to remember to thread through the implicit context during my rebase purgatory.  :smile: